src/pre_processing/extract_landscape.py

```diff
@@ -0,0 +1,190 @@
+import dlib
+import numpy as np
+import os
+import csv
+import logging
+
+DIR_PATH = os.path.dirname(os.path.realpath(__file__))
+PREDICTOR_PATH = DIR_PATH + "/shape_predictor_68_face_landmarks.dat"
+DATA_CSV_FILE = DIR_PATH + '/../../data/fer2013.csv'
+TEST_DATA_CACHED_PATH = DIR_PATH + "/saved_landscape_processing/landscape_test_data.npy"
+TRAIN_DATA_CACHED_PATH = DIR_PATH + "/saved_landscape_processing/landscape_train_data.npy"
+ALL_DATA_CACHED_PATH = DIR_PATH + "/saved_landscape_processing/landscape_all_data.npy"
+
+IMAGE_SIZE = 48 * 48
+
+TRAIN_END_POINT = 28708
+PUBLIC_TEST_START_POINT = 28709
+PUBLIC_TEST_END_POINT = 35887
+PRIVATE_TEST_END_POINT = 35887
+
+RIGHT_BROW = list(range(17, 22))
+LEFT_BROW = list(range(22, 27))
+NOSE = list(range(27, 35))
+RIGHT_EYE = list(range(36, 42))
+LEFT_EYE = list(range(42, 48))
+MOUTH = list(range(48, 61))
+
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p',
+                    level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+logger.addHandler(ch)
+
+
+def show_example():
+    """
+    Prints vectors found for first photo in file and shows picture with vectors painted on photo
+    """
+    photos_matrix = _extract_photos_from_file(DATA_CSV_FILE, True)
+    detector = dlib.get_frontal_face_detector()
+    predictor = dlib.shape_predictor(PREDICTOR_PATH)
+    img = photos_matrix[0, :, :]
```

Comment:
i just realised something - that syntax isn't even default python, looks like a numpy twist. I hereby do what i like to call "the timetabling" and recall my previous statement - in my head that was `photos_matrix[0][:][:]`.